libplatsupport/CMakeLists.txt

```diff
@@ -34,15 +34,29 @@ config_choice(
     "ega;LibPlatSupportX86ConsoleDeviceEGA;LIB_PLAT_SUPPORT_SERIAL_TEXT_EGA;KernelPlatPC99"
 )
 
+set(LibPlatSupportMach "")
+if(KernelPlatformRpi3)
```

Comment:
We could merge these to avoid some more redundancy
```
if(KernelPlatformRpi3 OR KernelPlatformRpi4)
    set(LibPlatSupportMach "bcm283x")
...
```

Reply:
Makes sense, corrected it.